exercises/practice/poker/.meta/config.json

```diff
@@ -1,6 +1,29 @@
 {
   "blurb": "Pick the best hand(s) from a list of poker hands.",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/0583fc6d09a4999ca032692018a977f06c052ee9 by Chris Doggett